mogwai/src/component.rs

```diff
@@ -232,36 +232,10 @@ where
       T::update(&mut t, msg, &tx_view, &subscriber);
     });
 
-    let out_msgs = Rc::new(RefCell::new(vec![]));
     rx_view.respond(move |msg: &T::ViewMsg| {
-      let should_schedule =
-        {
-        let mut msgs = out_msgs.borrow_mut();
-        msgs.push(msg.clone());
-        // If there is more than just this message in the queue, this
-        // responder has already been run this frame and a timer has
-        // already been scheduled, so there's no need to schedule another
-        msgs.len() == 1
-      };
-      if should_schedule {
-        let out_msgs_async = out_msgs.clone();
-        let tx_out_async = tx_out.clone();
-        utils::timeout(0, move || {
```

Comment:
I tried just replacing `utils::timeout` with `utils::set_immediate` and keeping the buffering, but performance was the same, so I kept the simpler version.

Reply:
I had at one point thought that I could use this queue to detect cycles that may lock up the UI. I was never successful at that though so I think this is a good simplification, especially if it helps performance. My only worry is that loops/cycles would be tighter with `set_immediate` and lock up the UI. Can you speak to that?

Reply:
After reading a bit more about `set_timeout` I'm satisfied (in theory) that it's snappy enough.

Reply:
> loops/cycles would be tighter with set_immediate and lock up the UI

Hmm, that might be possible, because this implementation synchronously runs callbacks that are scheduled during another callback. Avoiding this doesn't hurt performance, PR incoming.